agent/hcp/deps.go

```diff
@@ -86,5 +88,7 @@ func sink(hcpClient hcpclient.Client, cfg hcpclient.CloudConfig, logger hclog.Lo
 		return nil
 	}
 
+	logger.Info("Initialized HCP Metrics Sink")
```

Comment:
can this be debug? Also is there a standard in Consul logging about casing? I think generally the pattern is all lowercase except acronyms